### PR TITLE
Bug/159

### PR DESCRIPTION
**Pull request by theflexbird**
#159

### CODE REVIEW

File: src/client/constants/campaigns.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ export const CONTACT_LOAD_SUCCESS = 'CONTACT_LOAD_SUCCESS';
 export const CONTACT_LOAD_ERROR = 'CONTACT_LOAD_ERROR';
 
 export const CAMPAIGN_FIELDS = [
+  'id',
```
Comment (gr4per) [inline src/client/constants/campaigns.js]: this should have been removed as per our discussion

File: src/server/db/data/campaignContacts.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "email": "supplier@example.com",
-    "campaignId": "ncc-sob-w1",
+    "campaignId": 1,
```
Comment (gr4per) [inline src/server/db/data/campaignContacts.json]: campaignId is a string, not a number

File: src/server/db/data/campaignContacts.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
   },
   {
     "email": "karan@cronj.com",
-    "campaignId": "ncc-sob-w1",
+    "campaignId": 1,
```
Comment (gr4per) [inline src/server/db/data/campaignContacts.json]: campaignId is a string, not a number

File: src/server/routes/campaignContact.js [hunk 4]
```diff
@@ -11,17 +11,48 @@ module.exports = function(epilogue, db) {
...
+          return new Promise(
+            function(resolve, reject) {
+              db.models.Campaign.findOne({
+              where: {
+                campaignId: req.params.campaignId,
+                customerId: userData.customerid
+              }
+            }).then((campaign) => {
+              req.body.campaignId = campaign.id;
+              console.log(req.body);
```
Comment (patrykkopycinski) [inline src/server/routes/campaignContact.js]: not necessary

File: src/server/workflow/workflow.js [hunk 5]
```diff
@@ -89,13 +90,14 @@ module.exports = function(app, db) {
   /*
      API to load onboarding page
    */
-  app.get('/public/landingpage/:tenantId/:campaignId/:contactId', (req, res) => {
+  app.get('/public/landingpage/:tenantId([c|s]_[\w]{1,})/:campaignId/:contactId', (req, res) => {
```
Comment (patrykkopycinski) [inline src/server/workflow/workflow.js]: export regex to the variable

File: src/server/db/data/campaignContacts.json (hunk 2 above)
Comment (theflexbird) [inline src/server/db/data/campaignContacts.json]: It is a string but in different table.

File: src/server/db/data/campaignContacts.json (hunk 3 above)
Comment (theflexbird) [inline src/server/db/data/campaignContacts.json]: It is a string but in different table.

File: src/server/workflow/workflow.js (hunk 5 above)
Comment (theflexbird) [inline src/server/workflow/workflow.js]: What we are gaining doing it ?

**Pull request merged**